code/modules/heavy_vehicle/mech_interaction.dm

```diff
@@ -565,3 +542,154 @@
 		src.visible_message("<span class='warning'>\The [src] beeps loudly as its servos sieze up, and it enters lockdown mode!</span>")
 	else
 		src.visible_message("<span class='warning'>\The [src] hums with life as it is released from its lockdown mode!</span>")
+
+/mob/living/heavy_vehicle/get_floating_chat_x_offset()
+	return 8
+
+/mob/living/heavy_vehicle/hear_say(var/message, var/verb = "says", var/datum/language/language = null, var/alt_name = "", var/italics = 0, var/mob/speaker = null, var/sound/speech_sound, var/sound_vol)
+	if(can_listen())
+		addtimer(CALLBACK(src, .proc/handle_hear_say, speaker, message), 0.5 SECONDS)
```

Comment:
Why? You should just invoke_async this if the timer's going to be unnoticeably small anyway.

Reply:
spoke about it, it's just for aesthetic purposes since it's weird to reply INSTANTLY